packages/synapse-react-client/src/components/SynapseTable/TopLevelControls/TopLevelControlsUtils.test.ts

```diff
@@ -3,32 +3,44 @@ import {
   getNumberOfResultsToInvokeAction,
   getNumberOfResultsToInvokeActionCopy,
 } from './TopLevelControlsUtils'
-import { QueryResultBundle, Row } from '@sage-bionetworks/synapse-types'
+import { Row } from '@sage-bionetworks/synapse-types'
 
 describe('TopLevelControlsUtils', () => {
   describe('getNumberOfResultsToInvokeAction', () => {
     it('has selected rows', () => {
       const hasSelectedRows = true
       const selectedRows = [{}, {}] as Row[]
-      const data = { queryCount: 200 } as QueryResultBundle
+      const queryCount = 200
```

Comment:
queryCount is now passed as an argument instead of passing an entire QueryResultBundle